src/styles/main.scss

```diff
@@ -1 +1,66 @@
-// put code here
+@import "./utils/mixins";
+@import "./utils/vars";
+
+@keyframes rotate {
+  100% {
+    transform: rotateZ(360deg);
+  }
+}
```

Comment:
separate

Reply:
I am not sure what you meant by separate, so I added it from to and moved it to stopwatch block